hazelcast/config.py

```diff
@@ -1,8 +1,3 @@
-"""
-Hazelcast Client Configuration module contains configuration classes and various constants required to create a ClientConfig.
```

Comment:
Don't we need this doc comment here? Or it was redundant?

Reply:
I don't think we need this. It contains no useful information and the `ClientConfig` is not there anymore. The configuration classes also do not exist anymore and, their documentation is moved to the HazelcastClient's constructor.

Reply:
Makes sense